CMakeLists.txt

```diff
@@ -58,13 +58,9 @@ check_include_file (sys/types.h HAVE_SYS_TYPES_H)
 check_include_file (sys/stat.h HAVE_SYS_STAT_H)
 
 # eigen
-find_package (Eigen3 NO_MODULE)
-if (EIGEN3_FOUND)
-  include (${EIGEN3_USE_FILE})
-else ()
-  find_package (Eigen3 REQUIRED)
-  include_directories (${EIGEN3_INCLUDE_DIRS})
-endif()

```

Comment:
you're disabling the module features :(

Reply:
This was not working (i.e., I could not select the Eigen version that I wanted) in my setups (I tried in 2-3 different) and that's why I removed it. If @beniz agrees, you can make a PR to get it back (does not really hurt my experiments/setups -- I can always just have one Eigen installed or maybe you can give me a small example on how to use it).

Reply:
you should point to the eigen version using the path to its config file

Reply:
How do I point to the config file? And why is it different than setting `-DEIGEN3_INCLUDE_DIR` to your Eigen folder?

Reply:
use Eigen3_DIR to point to the eigen-config.cmake, this does not require the findeigen script anymore.